.github/workflows/create-apt-repo.yml

```diff
@@ -17,9 +17,9 @@ jobs:
       matrix:
         category:
           - name: stable
-            ref: 1.3.0  # Previous stable branch + backport
+            ref: v1.3.3  # Previous stable branch + backport
           - name: stable
-            ref: v1.4.1
+            ref: v1.4.5
```

Comment:
1.4.5 doesn't exist yet, target 1.4.4. I'm going to do 1.4.5 by end of this week though.